### DISCUSSION TITLE
Select GPU for training

### DISCUSSION THREAD

**Discussion by XanomalieX** [SWivid/F5-TTS#424] [Q&A] [answered]
I got 2 GPU and the slower one is auto selected for finetuning/training
Where can I select the right one?

Can't find it in code.
thanx

**Comment by sarpba** [comment]
```
accelerate config
```
![Képernyőkép 2024-11-08 00-23-54](https://github.com/user-attachments/assets/a7289ac7-04d4-4049-a1a0-9acd40043e4d)

0 or 1 dipend on whitch is your faster GPU.

**Comment by XanomalieX** [comment, marked as answer]
this was the solution for me:

CUDA_VISIBLE_DEVICES="0" python ./src/f5_tts/train/finetune_gradio.py
